Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -238,7 +238,7 @@ protected override void ActivateImpl(ActivateEventArgs args)
             }
         }
 
-        private enum Wires
+        private enum Wires : byte
```

Comment:
I can't see all the values on github but it's probably fine.

Reply:
Values goes form 0 to 6, so I think that 'byte' type will be good.